### PR TITLE
Add Delphi style guide

### PR DESCRIPTION
**Pull request by AlexP11223**

### CODE REVIEW

**Comment by Kristories** [comment]
Hi, @AlexP11223 thanks!

Do you think it's better to have a "Delphi" section than to put a link in a "Pascal" section? Give me a hint

**Comment by AlexP11223** [comment]
dunno, I guess one option is to make one "Pascal/Delphi/Lazarus" section. Because Pascal is mainly used in Delphi/Lazarus nowadays.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,10 @@ A set of guidelines for a specific programming language that recommend programmi
 
 - [Effective Dart](https://www.dartlang.org/guides/language/effective-dart)
 
+### Delphi
+
+- [Delphi’s Object Pascal Style Guide](https://docwiki.embarcadero.com/RADStudio/Alexandria/en/Delphi%E2%80%99s_Object_Pascal_Style_Guide)
```
Comment (Kristories) [inline README.md:62]: Can you change `’` to `'`

```
"’" is used without matching "‘"  remark-lint:match-punctuation
```

**Review by Kristories — CHANGES_REQUESTED** [review]

**Comment by Kristories** [comment]
I will merge after you change `Delphi’s` to `Delphi's`.

>  I guess one option is to make one "Pascal/Delphi/Lazarus" section. Because Pascal is mainly used in Delphi/Lazarus nowadays.

I still receive feedback from people.

File: README.md (hunk 1 above)
Comment (AlexP11223) [reply README.md]: Fixed.

**Review by Kristories — APPROVED** [review]

**Comment by Kristories** [comment]
👍🏼  LGTM. Thanks for your contribution! I'll deploy this as soon as CI is.

**Pull request merged**